### PR TITLE
[UK] Add `Bot Account`

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Bot_account/uk.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Основна відмінність особистих акаунтів від бот акаунтів - це ліміти які застосовуються до них при відправці повідомлень у [чаті](/wiki/Client/Interface/Chat_console). Бот акаунти мають більші ліміти, щоб дозволити їх сервісам взаємодіяти з більшою кількістю користувачів без ризику для акаунту отримати [сайленс](/wiki/Silence):
+
+- Особисті акаунти можуть надсилати 10 повідомлень кожні 5 секунд
+- Бот акаунти можуть надсилати 300 повідомлень кожні 60 секунд
+
+Ці ліміти стосуються тільки приватних повідомлень, `#multiplayer`, та `#spectator`. Бот акаунтам заборонено відправляти повідомлення в інших каналах, або будь-які небажані приватні повідомлення.
+
+## Створення бот акаунту
+
+Якщо ви хочете створити чат-бота, будь ласка, почніть з його запуску на вашому власному акаунті osu!, і обов'язково слідуйте лімітам персональних акаунтів у списку вище. По мірі того як ваш сервіс зростатиме, може з'явитися можливість запросити від команди підтримки акаунтів бот аккаунт для нього.
```
Comment (JyJyJ4JyJ) [inline wiki/Bot_account/uk.md]:
```suggestion
Якщо ви хочете створити чат-бота, будь-ласка, почніть з його запуску на вашому власному акаунті osu!, і обов'язково слідуйте лімітам персональних акаунтів у списку вище. З розширенням вашого сервісу у вас буде можливість відправити запит команді підтримки акаунтів для отримання окремого бот аккаунту.

```

File: wiki/Bot_account/uk.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Бот слідує лімітам персональних акаунтів
+- Бот не надсилає жодних повідомлень у публічних каналах
+- Бот не надсилає жодних небажаних приватних повідомлень
+- Бот корисний для широкої аудиторії в спільноті osu!
+
+### Подання запиту
+
+Якщо ваш бот відповідає наведеним вище критеріям, ви можете подати запит, щоб створити бот акаунт для нього.
+
+Надішліть електронний лист до [accounts@ppy.sh](mailto:accounts@ppy.sh) з назвою `Bot Account Request`. Він має бути відправлений з електронної пошти прив'язаної до вашого акаунту osu!.
```
Comment (JyJyJ4JyJ) [inline wiki/Bot_account/uk.md]:
```suggestion
Надішліть електронний лист до [accounts@ppy.sh](mailto:accounts@ppy.sh) з назвою `Bot Account Request`. Він має бути відправлений з електронної пошти, прив'язаної до вашого акаунту osu!.

```

File: wiki/Bot_account/uk.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Текст електронного листа має містити наступне:
+
+- Ваш нікнейм osu! (на якому працював ваш бот)
+- Нікнейм який ви хочете дати бот акаунту
```
Comment (JyJyJ4JyJ) [inline wiki/Bot_account/uk.md]:
```suggestion
- Нікнейм, який ви хочете дати бот акаунту

```

File: wiki/Bot_account/uk.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- Дата коли ваш бот почав працювати
```
Comment (JyJyJ4JyJ) [inline wiki/Bot_account/uk.md]:
```suggestion
- Дата, коли ваш бот почав працювати

```

File: wiki/Bot_account/uk.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- Посилання на код і документацію
+- Короткий опис того, що робить ваш бот
+
+[Команда підтримки акаунтів](/wiki/People/Account_support_team) перевірить ваш запит. Якщо ваш запит буде відхилено, вони дадуть вам знати, чому. Якщо ваш запит буде схвалено, вони створять бот акаунт для вас та нададуть вам подальші інструкції щодо його використання.
```
Comment (JyJyJ4JyJ) [inline wiki/Bot_account/uk.md]:
```suggestion
[Команда підтримки акаунтів](/wiki/People/Account_support_team) перевірить ваш запит. Якщо його буде відхилено, вони дадуть вам знати чому. Якщо ваш запит буде схвалено, вони створять бот акаунт для вас та нададуть вам подальші інструкції щодо його використання.

```

File: wiki/Bot_account/uk.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+## Дрібниці
+
+- Значок групи `BOT` показується на акаунтах з первинною групою `Chat Bots` (група #29). Список групи не є публічним.
+- Деякі персональні акаунти знаходяться в групі чат ботів через те, що власники не хотіли або не запросили окремий бот акаунт для їх сервісу, наприклад ::{ flag=DE }:: [Tillerino](https://osu.ppy.sh/users/2070907) та ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454). Це налаштування більше не пропонується, та нові боти, які приєднуються до групи, завжди отримують ексклюзивний акаунт.
```
Comment (JyJyJ4JyJ) [inline wiki/Bot_account/uk.md]:
```suggestion
- Деякі персональні акаунти знаходяться в групі чат ботів через те, що власники не хотіли або не запросили окремий бот акаунт для їх сервісу, наприклад, ::{ flag=DE }:: [Tillerino](https://osu.ppy.sh/users/2070907) та ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454). Це налаштування більше не пропонується, та нові боти, які приєднуються до групи, завжди отримують ексклюзивний акаунт.

```

**Review by JyJyJ4JyJ — CHANGES_REQUESTED** [review]

**Review by JyJyJ4JyJ — APPROVED** [review]

File: wiki/Bot_account/uk.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+---
+tags:
```
Comment (TicClick) [inline wiki/Bot_account/uk.md:2]: original tags must also be preserved

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**